test/pocketbase-types-example.ts

```diff
@@ -222,19 +223,57 @@ export type CollectionResponses = {
 	users: UsersResponse
 }
 
+// Utility types for create/update operations
+
+// Create type for Auth collections
+export type CreateAuth<T> = {
+	id?: RecordIdString
+	email: string
+	emailVisibility?: boolean
+	password: string
+	passwordConfirm: string
+	verified?: boolean
+} & Omit<{
+	[K in keyof T as T[K] extends IsoAutoDateString ? never : K]: T[K]
+}, 'id'>
+
+// Create type for Base collections
+export type CreateBase<T> = {
+	id?: RecordIdString
+} & Omit<{
+	[K in keyof T as T[K] extends IsoAutoDateString | (IsoAutoDateString | undefined) ? never : K]: T[K]
```

Comment:
Why is this line different for Auth and Base? 
Does it also need to be omitted from the update fields?

Could it be a common helper like this? (not tested)

```
// Autodate fields are not required in updates, because they are set automatically by the server
// File fields are stored as filename strings, but the File object is required when creating/updating.
export type FixTypesForUpdate<T> = {
  [K in keyof T as 
    T[K] extends IsoAutoDateString 
      ? never 
      : K
  ]: T[K] extends FileNameString 
      ? File 
      : T[K] extends FileNameString[] 
        ? File[] 
        : T[K]
}
```